test/integration-test.sh

```diff
@@ -406,4 +594,44 @@ log "Waiting 70 seconds for cool-off period"
 sleep 70
 
 # verify channel has been closed
-close_channel 1 5 "${api1}" "${addr5}" "true"
+close_channel 1 5 "${api1}" "${addr5}" "outgoing" "true"
+
+log "Waiting 120 seconds for channels to completely close"
+sleep 120
+
+test_get_all_channels() {
+  local node_api=${1}
+
+  channels=$(get_all_channels ${node_api} false)
+  channels_count=$(echo ${channels} | jq '.outgoing | length')
+
+  channels_with_closed=$(get_all_channels ${node_api} true)
+  channels_with_closed_count=$(echo ${channels_with_closed} | jq '.outgoing | length')
+
+  [[ "${channels_count}" -ge "${channels_with_closed_count}" ]] && { msg "There should be more channels returned with includeClosed flag: ${channels_count} !< ${channels_with_closed_count}"; exit 1; }
+  [[ "${channels_with_closed}" != *"Closed"* ]] && { msg "Channels fetched with includeClosed flag should return channels with closed status: ${channels_with_closed}"; exit 1; }
+  echo "Get all channels successfull"
+}
+
+test_get_all_channels ${api1}
+
+test_strategy_setting() {
```

Comment:
I would not want to add this to the smoke tests and instead have a separate smoke test setup for testing strategies.

Reply:
@tolbrino IMO lets keep this on and open an issue to move this into a separate test?